### PR TITLE
fix: Fix MSVC warnings

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

**Review by jserv — CHANGES_REQUESTED** [review]
Rebase the latest master branch.

File: sse2neon.h [hunk 1]
```diff
@@ -8636,9 +8636,9 @@ FORCE_INLINE __m128i _mm_aesenc_si128(__m128i a, __m128i RoundKey)
 #define SSE2NEON_AES_B2W(b0, b1, b2, b3)                 \
     (((uint32_t) (b3) << 24) | ((uint32_t) (b2) << 16) | \
      ((uint32_t) (b1) << 8) | (uint32_t) (b0))
-// muliplying 'x' by 2 in GF(2^8)
+    // muliplying 'x' by 2 in GF(2^8)
```
Comment (jserv) [inline sse2neon.h:8639]: Don't change this.

File: sse2neon.h [hunk 2]
```diff
@@ -8636,9 +8636,9 @@ FORCE_INLINE __m128i _mm_aesenc_si128(__m128i a, __m128i RoundKey)
...
 #define SSE2NEON_AES_F2(x) ((x << 1) ^ (((x >> 7) & 1) * 0x011b /* WPOLY */))
-// muliplying 'x' by 3 in GF(2^8)
+    // muliplying 'x' by 3 in GF(2^8)
```
Comment (jserv) [inline sse2neon.h:8641]: Ditto. Minimize the necessary changes.

**Comment by howjmay** [comment]
I think the rest warnings should be false positive

**Pull request merged**